src/humanize/time.py

```diff
@@ -257,9 +257,19 @@ def precisetime(
         delta, minimum_unit, suppress=suppress, format=format, when=now
     )
 
-    # janky but probably relatively efficient to check for "0 [any unit]"
-    splitdelta = delta.split(" ")
-    if len(splitdelta) == 2 and float(splitdelta[0]) == 0:
+    # this line addresses a theoretical edge case where a language doesnt put spaces before/after numbers
+    splitdelta = "".join(
+        i for i in delta if i.isdigit() or i in " ,."
```

Comment:
Rather than trying to figure out if the value of the processed string `delta` is zero, can we check the original `delta` returned by `date_and_delta`? That's a `datetime.timedelta`.

Reply:
Problem with that is that it doesn’t take into account the minimum/suppressed units passed to precisedelta

Reply:
Can someone explain why it is appropriate to replace `0` with "now" for a precise time? I think that only natural time should "now" because "now" is imprecise.

Reply:
I think there's 2 reasons: I initially put it in there to replicate the behavior of natural time and because returning either "0 seconds _from now_" or "0 seconds _ago_" feels strange to me and more like it's bugging out or rounding improperly 

I don't really mind if this changes, I'm just explaining why it's there

Reply:
But if you get the string "0 seconds from now", that's what you asked for by specifying the `minimum_unit` and `format` parameter? If you wanted something like "0.12 seconds from now", then you need to ask for higher precision?

Reply:
Maybe we could put an optional parameter to specify if the user wants "now" or "0 [unit] ago/from now"